mkdocs.yml

```diff
@@ -219,7 +219,7 @@ extra:
         kafkaversion: 2.6
         kafkarelease: 5.5.0-ccs
```

Comment:
Also, I was wondering yesterday if these Kafka and CP versions are still correct...

Reply:
I don't see usages of kafkarelease in our repo but I'd guess we should be on 6.0 now instead of 5.5? Let me merge this PR as is since I'll need to cherry-pick to 0.12. Happy to open another if you think updating this variable makes sense.

Reply:
Thanks for the deets, I'll go ahead and update.